src/conf/data/partial_label.yaml

```diff
@@ -0,0 +1,37 @@
+# @package _global_
+data:
+  batch_size: 64
+  num_workers: 10
+  sample_rate: 16000
+  data_sample_rate: 44100
+  train_dataset:
+    _target_: data.ClassifierDataset
+    data_path: '../../finding_willy/recordings_2018_filtered/'
+    mode: train 
+    metadata_path: '../annotations/combined_annotations_filtered_train.csv'
+    augmentations_p: 0.8
+    augmentations: ${_augmentations}
+    preprocessors: ${_preprocessors}
+    seq_length: 0.5
+    len_buffer: 0.1
+    margin_ratio: 0.5
```

Comment:
Sorry for silly question - but doesn't this behavior kinda overlap with partial positives? Can you remind us what the margin_ratio does?